jupyterdrive/gdrive/drive-contents.js

```diff
@@ -113,7 +113,11 @@ define(function(require) {
             var metadata = values[0];
             var contents = values[1];
             var model = files_resource_to_contents_model(path, metadata);
-            model['content'] = notebook_model.notebook_from_file_contents(contents);
+            if(metadata.mimetyme == drive_utils.NOTEBOOK_MIMETYPE){
```

Comment:
type seem to have been dropped from main IPython. I'll drop it too.

file_resource_to_contents_model seem to deal only with metadata, I can add the content as an optional arg, it makes sens.